### ISSUE TITLE
Lint error on CI = BUILD FAILED, but at the end SUCCESS

### ISSUE THREAD

**Issue by bekuno** [cgeo/cgeo#10730]
## Bug description 
If a build failed, then the final state should be ERROR.
Here we have a failed build for an lint error, bur state SUCCESS.

### Steps to reproduce the problem
see https://ci.cgeo.org/job/cgeo%20pull%20request/5894/console
see https://ci.cgeo.org/job/cgeo%20pull%20request/5833/console
~~~
> Task :main:lintBasicDebug FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':main:lintBasicDebug'.
> Lint found errors in the project; aborting build.
...
BUILD FAILED in 5m 13s
....
Finished: SUCCESS
~~~

**Comment by eddiemuc** [comment]
Is this something to reconfigure on jenkins?
I am currently at a loss why jenkins marks the build as "green" when the build failed...

**Comment by bekuno** [comment]
This is a problem on jenkins.
I dont know for now if we have to configure something or if this is a bug and on which component.

**Comment by xmichele** [comment]
Hello,

as anticipated via e-mail, I'd like to contribute to this.
How can I obtain credentials to operate on https://ci.cgeo.org/ ? 

Thanks!

**Comment by Lineflyer** [comment]
Hi @xmichele 

I think we were in contact via our support mail address before.

The access to our CI uses OAuth from Github, so that c:geo team members (with write access to the repo) also have access to the CI. I am not sure how we handle other access requests.

You have to understand, that the CI is a very important tool for us, so we do not want to provide access instantly to a new contributor. But - of course - helping hands are always welcome, especially as our CI might need some polishing.

Not sure about the best way to get to that point. I would suggest you might use the "discussions" area of this repo to introduce yourself a bit and share your background and ideas for our CI.
The team members with most experience of our CI are @mucek4 and @kumy (I guess), so you could start with some questions towards them.

**Comment by xmichele** [comment]
Sure.
In the meantime I did 2 commits for this and I think they are in the queue fro the merge

**Comment by xmichele** [comment]
@Lineflyer please check if I have permissions for a new discussion.  Thanks :)

**Comment by Lineflyer** [comment]
> @Lineflyer please check if I have permissions for a new discussion. Thanks :)

I could not find a setting for that on github.
Are you sure you can't open a new one here: https://github.com/cgeo/cgeo/discussions

**Comment by xmichele** [comment]
Basically I have no "New Discussion" or "Settings" buttons on this discussion tab

**Comment by Lineflyer** [comment]
@xmichele For some reason this seems to be limited to team members. I started a topic for you to reply to:
https://github.com/cgeo/cgeo/discussions/11365

**Comment by bekuno** [comment]
The problem should be fixed by the changes of @kumy in #12182, esp. see https://github.com/cgeo/cgeo/issues/12182#issuecomment-981116671 .
